lib/blocks.php

```diff
@@ -93,3 +93,93 @@ function do_blocks( $content ) {
 	return $content_after_blocks;
 }
 add_filter( 'the_content', 'do_blocks', 9 ); // BEFORE do_shortcode() and wpautop().
+
+/**
+ * Extract the blocks from post content for the REST API post response.
+ *
+ * @since 1.1.0
+ *
+ * @param string $content The post content.
+ *
+ * @return array Array of block data.
+ */
+function gutenberg_add_blocks_to_post_resource( $content ) {
```

Comment:
This function name doesn't _quite_ match up with what it's doing. I'd suggest something like `gutenberg_extract_blocks_from_post_content`.

Reply:
Right, good suggestion.

Reply:
updated